host-bmc/dbus/serialize.cpp

```diff
@@ -140,8 +143,9 @@ void Serialize::reSerialize(const std::vector<uint16_t> types)
     {
         if (savedObjs.contains(type))
         {
-            std::cerr << "Removing objects of type : " << (unsigned)type
-                      << " from the persistent cache\n";
+            error(
```

Comment:
This should be an info log, it can go in as a separate commit.

Reply:
Sure i will make a PR for improvision of logs